core/domain/feedback_services.py

```diff
@@ -37,9 +37,21 @@
 def _create_models_for_thread_and_first_message(
         exploration_id, state_name, original_author_id, subject, text,
         has_suggestion):
-    """Creates a thread and the first message in it.
+    """Create a feedback thread and its first message
 
-    Note that `state_name` may be None.
+    Args:
+        exploration_id: str. The exploration id the thread belongs.
+        state_name: str or None. The state name of the thread
+            which could be None.
```

Comment:
The state name for the thread. If None, this indicates that the thread pertains to the exploration as a whole.